apps/server/src/modules/room/domain/do/room.do.ts

```diff
@@ -0,0 +1,72 @@
+import { AuthorizableObject, DomainObject } from '@shared/domain/domain-object';
+import { EntityId } from '@shared/domain/types';
+
+export interface RoomProps extends AuthorizableObject {
+	id: EntityId;
+	name: string;
+	color: string;
+	startDate?: Date;
+	untilDate?: Date;
+	createdAt?: Date;
+	updatedAt?: Date;
+}
```

Comment:
The timestamps shouldn't be optional. They are defined as soon as the DO starts to exist.